index.html

```diff
@@ -57,7 +57,7 @@ <h1 class="title">I'm Shaheda <br>Glad to see you!
         </section>
     </div>
 
```

Comment:
- [ ] I really salute your hard work, so professional, however, there are a few things we can change, take a look are your page title bellow, it is not really descriptive, don't you think like me? making it something like `Shahe's portfolio` would be perfect and more descriptive, please make this change. 
- [ ] 
<img width="291" alt="tit" src="https://user-images.githubusercontent.com/79058364/156215246-f7f36a5d-5a10-4076-85fa-3d53a3f7b1fe.png">

- [ OPTIONAL] Link make your nav links functional, see this image below, am supposed to be on the about section when I click the about nav link but nothing happened, please fix this, it will prevent to the user a long scroll.  
- 
<img width="782" alt="link" src="https://user-images.githubusercontent.com/79058364/156215664-367e5c65-426f-4488-8cd0-befad738d51b.png">